rtgui/histogrampanel.cc

```diff
@@ -994,10 +961,10 @@ void HistogramArea::drawCurve(Cairo::RefPtr<Cairo::Context> &cr,
     double s = RTScalable::getScale();
 
     cr->set_line_width(s);
-    cr->move_to (padding, vsize - 1);
+    //cr->move_to (padding, vsize - 1);
     scale = scale <= 0.0 ? 0.001 : scale; // avoid division by zero and negative values
 
-    for (int i = 0; i < 256; i++) {
+    for (int i = 0; i < 65536; i++) {
```

Comment:
@heckflosse This could be an additional bottleneck, but since it's related to GUI drawing, it's probably not parallellizable.

Reply:
If each line averages 50 pixels long, that's 65536 x 50 x 3 = about 10 million pixels of line for the raw histogram. That's a lot of line...

Reply:
I placed a `BENCHFUN` in `HistogramArea::drawCurve`
Each time I change for example the exposure adjuster I get 3 timings of ~28 ms, means changing exposure adjuster needs 3 * 28 ms to update the histogram.

Questions:
1) why 3 timings instead of one?
2) I was in RAW histogram mode. Why is the histogram redrawn when I change the exposure adjuster?

I made a quick hack (still without using SSE), which reduces the time to 3 * 7 ms. Will try with SSE now...

Reply:
@heckflosse Feel free to push to this branch whenever you feel it's useful!